{"uid":"b72186cbb575bf6","name":"Successful Login","fullName":"Login functionallity: Successful Login","historyId":"6eb4afeab05f2b214d8d7d19c20f537e","time":{"start":1732868838232,"stop":1732868846772,"duration":8540},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//H1[text()='Today']\"}\n  (Session info: chrome=131.0.6778.70); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n\tGetHandleVerifier [0x00007FF725C92775+28773]\n\t(No symbol) [0x00007FF725BFAFB0]\n\t(No symbol) [0x00007FF725A9552A]\n\t(No symbol) [0x00007FF725AE8EAE]\n\t(No symbol) [0x00007FF725AE919C]\n\t(No symbol) [0x00007FF725B323F7]\n\t(No symbol) [0x00007FF725B0EFDF]\n\t(No symbol) [0x00007FF725B2F1A4]\n\t(No symbol) [0x00007FF725B0ED43]\n\t(No symbol) [0x00007FF725ADA548]\n\t(No symbol) [0x00007FF725ADB6B1]\n\tGetHandleVerifier [0x00007FF725FBF45D+3358029]\n\tGetHandleVerifier [0x00007FF725FD430D+3443709]\n\tGetHandleVerifier [0x00007FF725FC83FD+3394797]\n\tGetHandleVerifier [0x00007FF725D5929B+842635]\n\t(No symbol) [0x00007FF725C0654F]\n\t(No symbol) [0x00007FF725C01FA4]\n\t(No symbol) [0x00007FF725C0213D]\n\t(No symbol) [0x00007FF725BF1629]\n\tBaseThreadInitThunk [0x00007FF92E897034+20]\n\tRtlUserThreadStart [0x00007FF930062651+33]\n\n","statusTrace":"  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"tests/ui/steps/login_steps.py\", line 23, in step_verify_login_successful\n    assert context.login_page.driver.find_element(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//H1[text()='Today']\"}\n  (Session info: chrome=131.0.6778.70); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n\tGetHandleVerifier [0x00007FF725C92775+28773]\n\t(No symbol) [0x00007FF725BFAFB0]\n\t(No symbol) [0x00007FF725A9552A]\n\t(No symbol) [0x00007FF725AE8EAE]\n\t(No symbol) [0x00007FF725AE919C]\n\t(No symbol) [0x00007FF725B323F7]\n\t(No symbol) [0x00007FF725B0EFDF]\n\t(No symbol) [0x00007FF725B2F1A4]\n\t(No symbol) [0x00007FF725B0ED43]\n\t(No symbol) [0x00007FF725ADA548]\n\t(No symbol) [0x00007FF725ADB6B1]\n\tGetHandleVerifier [0x00007FF725FBF45D+3358029]\n\tGetHandleVerifier [0x00007FF725FD430D+3443709]\n\tGetHandleVerifier [0x00007FF725FC83FD+3394797]\n\tGetHandleVerifier [0x00007FF725D5929B+842635]\n\t(No symbol) [0x00007FF725C0654F]\n\t(No symbol) [0x00007FF725C01FA4]\n\t(No symbol) [0x00007FF725C0213D]\n\t(No symbol) [0x00007FF725BF1629]\n\tBaseThreadInitThunk [0x00007FF92E897034+20]\n\tRtlUserThreadStart [0x00007FF930062651+33]\n\n","statusTrace":"  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"tests/ui/steps/login_steps.py\", line 23, in step_verify_login_successful\n    assert context.login_page.driver.find_element(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[{"name":"Given the user navigates to the login page","time":{"start":1732868838233,"stop":1732868840824,"duration":2591},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"When the user logs in with valid credentials","time":{"start":1732868840824,"stop":1732868841681,"duration":857},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Then the user should be redirected to the dashboard","time":{"start":1732868841681,"stop":1732868846771,"duration":5090},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//H1[text()='Today']\"}\n  (Session info: chrome=131.0.6778.70); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n\tGetHandleVerifier [0x00007FF725C92775+28773]\n\t(No symbol) [0x00007FF725BFAFB0]\n\t(No symbol) [0x00007FF725A9552A]\n\t(No symbol) [0x00007FF725AE8EAE]\n\t(No symbol) [0x00007FF725AE919C]\n\t(No symbol) [0x00007FF725B323F7]\n\t(No symbol) [0x00007FF725B0EFDF]\n\t(No symbol) [0x00007FF725B2F1A4]\n\t(No symbol) [0x00007FF725B0ED43]\n\t(No symbol) [0x00007FF725ADA548]\n\t(No symbol) [0x00007FF725ADB6B1]\n\tGetHandleVerifier [0x00007FF725FBF45D+3358029]\n\tGetHandleVerifier [0x00007FF725FD430D+3443709]\n\tGetHandleVerifier [0x00007FF725FC83FD+3394797]\n\tGetHandleVerifier [0x00007FF725D5929B+842635]\n\t(No symbol) [0x00007FF725C0654F]\n\t(No symbol) [0x00007FF725C01FA4]\n\t(No symbol) [0x00007FF725C0213D]\n\t(No symbol) [0x00007FF725BF1629]\n\tBaseThreadInitThunk [0x00007FF92E897034+20]\n\tRtlUserThreadStart [0x00007FF930062651+33]\n\n","statusTrace":"  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"tests/ui/steps/login_steps.py\", line 23, in step_verify_login_successful\n    assert context.login_page.driver.find_element(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":3,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"severity","value":"normal"},{"name":"tag","value":"login"},{"name":"tag","value":"successful_login"},{"name":"feature","value":"Login functionallity"},{"name":"framework","value":"behave"},{"name":"language","value":"cpython3"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["successful_login","login"]},"source":"b72186cbb575bf6.json","parameterValues":[]}